.github/workflows/check-patch.yml

```diff
@@ -0,0 +1,105 @@
+name: Check patch
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build-el8:
+
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/centos/centos:stream8
+
+    steps:
+    - name: prepare env
+      run: |
+           mkdir -p ${PWD}/tmp.repos/BUILD
+           yum install -y --setopt=tsflags=nodocs autoconf automake gettext-devel git systemd make git rpm-build
```

Comment:
Why use 'yum' everywhere and not 'dnf'? 'yum' indicates you might have intentions to support also el7 (eventually), which I think you do not

Reply:
no strong feelings about using yum or dnf here as they are the same code being executed, I guess it's just I used to type yum instead of dnf :-)